src/registrar/models/domain.py

```diff
@@ -1062,6 +1062,15 @@ def is_expired(self):
         now = timezone.now().date()
         return self.expiration_date < now
 
+    def state_display(self):
+        """Return the display status of the domain."""
+        if self.is_expired() and self.state != self.State.UNKNOWN:
+            return "Expired"
+        elif self.state == self.State.UNKNOWN or self.state == self.State.DNS_NEEDED:
+            return "DNS needed"
```

Comment:
**non blocking comment**: I get why we are doing this... but could we use a constant here? I know this is already a losing battle as have so much intermingled content in code and html, but if we can use constants it does make life easier when changes come around.

Also I think DNS needed should use the domain state DN_NEEDED constant value and we should fix the typo in that state that makes it "Dns" instead of "DNS".

Reply:
We could get the state value, but what we need is the state **display** value for dns needed (and there's no const for expired). Do you know how to get a state_display_value for a a specific value as opposed to whatever's on the domain?